src/main/java/org/terasology/ReplayTestingEnvironment.java

```diff
@@ -56,17 +78,18 @@
      * @throws Exception
      */
     protected void openMainMenu() throws Exception {
-        host = createEngine();
+        host = createEngine(false);
         host.run(new StateMainMenu());
     }
 
     /**
-     * Opens the game in a replay.
-     * @param replayTitle the name of the replay to be played when the game opens.
+     * Opens the game in a replay state.
+     * @param replayTitle the title of the replay to be opened.
+     * @param isHeadless if the engine should be headless.
      * @throws Exception
      */
-    protected void openReplay(String replayTitle) throws Exception {
-        host = createEngine();
+    protected void openReplay(String replayTitle, boolean isHeadless) throws Exception {
```

Comment:
Should it also be pointed out that Jenkins (and/or other CI systems) may not be able to run headed replay tests and tests using headed replays should be `@Ignore`d?